Neos.Flow/Tests/Unit/Mvc/DispatcherTest.php

```diff
@@ -166,12 +151,11 @@ public function dispatchIgnoresStopExceptionsForFirstLevelActionRequests()
      */
     public function dispatchCatchesStopExceptionOfActionRequestsAndRollsBackToTheParentRequest()
     {
-        $this->mockActionRequest->expects(self::atLeastOnce())->method('isDispatched')->willReturn(false);
-        $this->mockParentRequest->expects(self::atLeastOnce())->method('isDispatched')->willReturn(true);
+        $this->markTestSkipped('TODO currently not implemented to state what it says. Should fail.');
```

Comment:
This is tricky, and might point to something we are breaking here that shouldn't be broken 🧐

Reply:
`AndRollsBackToTheParentRequest` mmmm

Reply:
my commit description says following ... maybe it helps :D

**_TASK: Remove obsolete "RollsBackToTheParentRequest" logic in dispatcher_**

As discussed we concluded that this logic might be faulty and actually do harm without the `isDispatched` in place:

Background:

the dispatcher is either used for main request, in which case this logic is not needed, or for subrequest like in a fusion plugin

Scenario:

A subrequest, while being handled, throws a `StopActionException`.
which is caught here. We check if it's a main request and get the main request which is now to be handled in the loop, like a forward.

Thesis:

Now it will actually render the outer response inside the plugin
But previously because the main request was dispatched === true, it would not be handled.

Example:

Using `throwStatus` in a plugin will result in the main request rendering being started again and again until the memory is exhausted.